src/rules/order.js

```diff
@@ -547,7 +551,12 @@ module.exports = {
                   type: 'string',
                   enum: ['after', 'before'],
                 },
+                insertNewline: {
+                  type: 'boolean',
+                  default: true,
```

Comment:
I’m not sure if json schema defaults actually apply in every eslint version. Separately, it’s better to have boolean options treat absence the same as false. Can we reword this so it can default to false?